$OpenBSD: patch-gnulib_tests_test-sys_time_c,v 1.1 2012/09/24 12:59:58 jasper Exp $

timeval.tv_sec is currently a long on OpenBSD.
Remove this when time_t becomes 64-bit.

--- gnulib/tests/test-sys_time.c.orig	Mon Sep 24 14:55:36 2012
+++ gnulib/tests/test-sys_time.c	Mon Sep 24 14:55:57 2012
@@ -24,7 +24,9 @@
 struct timeval a;
 
 /* Check that &a.tv_sec is a 'time_t *', ignoring signedness issues.  */
+#ifndef __OpenBSD__
 typedef int verify_tv_sec_type[sizeof (a.tv_sec) == sizeof (time_t) ? 1 : -1];
+#endif
 
 int
 main (void)
